static/hosted-doap/pade.doap

```diff
@@ -5,9 +5,319 @@
          >
```

Comment:
Please add ` xmlns:schema="https://schema.org/"` for the logo element.

Reply:
Done